holesky/README.md

```diff
@@ -20,7 +20,7 @@ NODE_INTERNAL_V2_RETRIEVAL_PORT=${NODE_V2_RETRIEVAL_PORT}
 
 ### 2. Update `MAIN_SERVICE_IMAGE`
 ```
-MAIN_SERVICE_IMAGE=ghcr.io/layr-labs/eigenda/opr-node:2.4.0-rc.1
+MAIN_SERVICE_IMAGE=ghcr.io/layr-labs/eigenda/opr-node:2.5.0
```

Comment:
@pschork The mainnet README and the hoodi README seem different, do you think we need to update the mainnet one or the hoodi one to match?